Document/Example2.ipynb

```diff
@@ -14,6 +14,13 @@
     "# Example-2 (How to plot via matplotlib)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "⚠️ This example is deprecated. you can use `plot` method from `version 3.0`"
```

Comment:
Please change it to:
"⚠️ This example is deprecated. You can use `plot` method from `version 3.0`"

Reply:
Fixed in https://github.com/sepandhaghighi/pycm/pull/344/commits/fe85fd4c1b8cde4f5e5f851b3a00fecbe8b7a0c6